javascript/grammar.json

```diff
@@ -954,7 +954,7 @@
             },
             "status": {
               "experimental": false,
-              "standard_track": false,
+              "standard_track": true,
```

Comment:
I set this as well, since it appears to be reflecting the same underlying fact.

Reply:
Should we add https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#sec-import-call-runtime-semantics-evaluation as a `spec_url` then?

Reply:
Done with 8ba99d2a7.

Reply:
I guess there's no reason not to do it for both features. I've added it with 52ed9465b.